var/spack/repos/builtin/packages/harfbuzz/package.py

```diff
@@ -8,9 +8,10 @@
 
 class Harfbuzz(AutotoolsPackage):
     """The Harfbuzz package contains an OpenType text shaping engine."""
-    homepage = "http://www.freedesktop.org/wiki/Software/HarfBuzz/"
-    url      = "http://www.freedesktop.org/software/harfbuzz/release/harfbuzz-0.9.37.tar.bz2"
+    homepage = "https://github.com/harfbuzz/harfbuzz"
+    git      = "https://github.com/harfbuzz/harfbuzz.git"
```

Comment:
You should still keep a `url` up here so that `spack versions` and `spack checksum` continue to work.

Reply:
Oh wow I didn't even realize there were tools to help add versions. I've been doing it all manually.